src/os_auth/local-server.c

```diff
@@ -342,6 +342,7 @@ char* local_dispatch(const char *input) {
     output = cJSON_PrintUnformatted(response);
     cJSON_Delete(response);
     cJSON_Delete(request);
+    os_free(groups);
```

Comment:
As we are now freeing groups here, we can avoid the extra **os_free(groups)** in line 286.